conbench/app/_plots.py

```diff
@@ -168,13 +169,21 @@ def _should_format(data, unit):
     return should, axis_unit
 
 
+def _insert_nans(some_list: list, indexes: List[int]):
+    """Insert np.NaNs into a list before the given indexes."""
+    for ix in sorted(indexes, reverse=True):
+        some_list.insert(ix, np.NaN)
+    return some_list
+
+
 def _source(
     data,
     unit,
     formatted=False,
     distribution_mean=False,
     alert_min=False,
     alert_max=False,
+    break_line_ixs: Optional[List[int]] = None,
```

Comment:
That I'm asking suggests we should have some docstrings, but what's `ixs`? "Indices"?

Reply:
Yes, good callout. I should have made this name a little more explicit